google-cloud-network_connectivity-v1/.repo-metadata.json

```diff
@@ -0,0 +1,15 @@
+{
+    "api_id": "networkconnectivity.googleapis.com",
+    "client_documentation": "https://googleapis.dev/ruby/google-cloud-network_connectivity-v1/latest",
+    "distribution_name": "google-cloud-network_connectivity-v1",
+    "language": "ruby",
+    "name": "networkconnectivity",
+    "name_pretty": "Network Connectivity V1 API",
+    "product_documentation": "https://cloud.google.com/network-connectivity/docs",
+    "repo": "googleapis/google-cloud-ruby",
+    "requires_billing": true,
+    "ruby-cloud-description": "Network Connectivity is Google's suite of products that provide enterprise connectivity from your on-premises network or from another cloud provider to your Virtual Private Cloud (VPC) network. Note that google-cloud-network_connectivity-v1 is a version-specific client library. For most uses, we recommend installing the main client library google-cloud-network_connectivity instead. See the readme for more details.",
+    "ruby-cloud-env-prefix": "NETWORK_CONNECTIVITY",
```

Comment:
Do you want `ruby-cloud-env-prefix` for this library?

Reply:
It's present for the existing v1alpha1 client, so we should have it here also for consistency.